packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -666,7 +698,13 @@ class InteractiveGraphEditor extends React.Component<Props> {
             // @ts-expect-error TS2339 Property 'getUserInput' does not exist on type 'ReactInstance'. Property 'getUserInput' does not exist on type 'Component<any, {}, any>'.
             const correct = graph && graph.getUserInput();
             _.extend(json, {
-                graph: {type: correct.type},
+                graph: {
+                    type: correct.type,
+                    coords:
+                        this.props.graph?.type === "circle"
```

Comment:
This is the serialization function, which is used to copy/paste the graph across editors. Before, `coords` was never being used, because it was never part of the initial state. It was always undefined, and then it got set as the user would start interacting with it.

In this PR, I'm trying to set initial specific coordinates on the graph, and it needs to be copy/pastable in that state, which is why I now needed to add `coords` here when it wasn't there before.

I'll add a comment here for clarity!